### ISSUE TITLE
Update JRE to 11.0.6 to fix minimize button on macOS

### ISSUE THREAD

**Issue by TylerADavis** [runelite/launcher#81]
The minimize button at the top of the UI currently isn't working in macOS (https://github.com/runelite/runelite/issues/2967), and it seems that this is actually an issue with the JVM. Namely, the iconify state does not work on JFrames if the frame is undecorated. 

See the following bug: https://bugs.openjdk.java.net/browse/JDK-8214046

This bug has been fixed in 11.0.6, so we should update the JRE for the Mac launcher to resolve the issue.

**Comment by devLotto** [comment]
Thank you, a newer JRE version does indeed seem to fix it. 

It will take some time before we can release a launcher with it fixed:

AdoptOpenJDK's 11.0.6 release and older weren't signed with hardened runtime enabled, so a launcher with that embedded won't currently notarize, meaning it won't be trusted by macOS Catalina. When we signed and notarized the launcher last time, exceptions were in place by Apple that enabled it to be notarized regardless of that, but those have since been removed. (https://developer.apple.com/news/?id=09032019a)

However, according to https://github.com/AdoptOpenJDK/openjdk-build/issues/1130#issuecomment-601274247, AdoptOpenJDK's build for the next CPU release (11.0.7) will be fully notarized. The 11.0.7 release will be available on April 14 according to https://wiki.openjdk.java.net/display/JDKUpdates/JDK11u, so I should be able to update the launcher and fix this issue soon after that.
